modules/admin-ui-picard/app/src/components/shared/modals/ResourceDetailsAccessPolicyTab.js

```diff
@@ -0,0 +1,457 @@
+
+                                                                                {/* Remove policy */}
+                                                                                { true /*Todo: show only if: $root.userIs('ROLE_UI_EVENTS_DETAILS_ACL_EDIT')*/ && (
+                                                                                    <td>
+                                                                                        { !transactions.read_only && (
+                                                                                            <a onClick={() => remove(index)} className="remove" />
+                                                                                        )}
+                                                                                    </td>
+                                                                                )}
+                                                                            </tr>
+                                                                        ))
+                                                                        }
+
+                                                                        {/* create additional policy */}
+                                                                        { !transactions.read_only /*Todo:   && $root.userIs('ROLE_UI_EVENTS_DETAILS_ACL_EDIT') */ && (
+                                                                            <tr>
+                                                                                <td colSpan="5">
+                                                                                    <a onClick={() => push(createPolicy(""))}>
+                                                                                        + {t("EVENTS.EVENTS.DETAILS.ACCESS.ACCESS_POLICY.NEW")}
+                                                                                    </a>
+                                                                                </td>
+                                                                            </tr>
+                                                                        )}
+                                                                    </>
+                                                                )}
+                                                            </FieldArray>
+                                                            </tbody>
+
+                                                        </table>
+                                                    </div>
+                                                </div>
+                                            </div>
+
+                                            {/* Save and cancel buttons */}
+                                            { (!transactions.read_only && policyChanged) && (
+                                                <footer style={{padding: '15px'}}>
```

Comment:
Only show the buttons, if the user actually changed something. Currently, the buttons are still shown after clicking the update button. You can check this using `formik.isValid` and `formik.dirty`.

Reply:
This is handled through _policyChanged_. The buttons were only still shown after pressing update, because the update wasn't successful so far (I hadn't added a mock-backend file for this. I have added one now and it should work now). I also forgot to change _policyChanged_ after the update was successful, this was changed now.